nixos/modules/config/xdg/icons.nix

```diff
@@ -1,23 +1,22 @@
 { config, lib, pkgs, ... }:
 
-with lib;
 {
   meta = {
-    maintainers = teams.freedesktop.members;
+    maintainers = lib.teams.freedesktop.members;
   };
 
   options = {
-    xdg.icons.enable = mkOption {
-      type = types.bool;
-      default = true;
+    xdg.icons.enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
```

Comment:
Generally, :+1: 

We can remove it from minimal profile then.

Also will probably need to add it Plasma.

And write a release notes entry so that people rolling their own DEs are not blindsided.

Reply:
> will probably need to add it Plasma.

did so in #327913, wouldn't mind a review :)

Reply:
> Also will probably need to add it Plasma.

nope, that is already done in nixos/modules/services/misc/graphical-desktop.nix

> And write a release notes entry so that people rolling their own DEs are not blindsided.

👍🏼